ext/debian/pe-bolt-server.init

```diff
@@ -8,8 +8,9 @@
 # Default-Stop:      0 1 6
 ### END INIT INFO
 
-export GEM_HOME=/opt/puppetlabs/server/apps/bolt-server/lib/ruby/gems/2.4.0
-exec=/opt/puppetlabs/server/apps/bolt-server/bin/bolt-server
+prefix='/opt/puppetlabs/server/apps/bolt-server'
+export GEM_HOME="${prefix}/lib/ruby/gems/2.4.0"
+exec="${prefix}/bin/bolt-server ${prefix}/config.ru"
```

Comment:
I think the service files will all need to add `#{prefix}/bin` to PATH so bolt-server can find rackup.